csm/core/services/security.py

```diff
@@ -311,16 +311,24 @@ async def _timer_task(self, handler, start: datetime, interval: timedelta):
             current = datetime.now(timezone.utc)
             start = start + interval
 
+    async def _local_timezone(self, from_time):
+        """
+        Convert utc aware time to local timezone and print in specific format
+        """
+        local_time = from_time.astimezone()
+        return local_time.strftime(const.CERT_TIME_FORMAT)
+
     async def _check_certificate_expiry_time(self, current_time):
         warning_days = Conf.get(const.CSM_GLOBAL_INDEX, "SECURITY.ssl_cert_expiry_warning_days")
         try:
             expiry_time = await self.get_certificate_expiry_time()
             expiry_time = expiry_time.replace(tzinfo=timezone.utc)
+            local_time = self._local_timezone(expiry_time)
```

Comment:
Thisi s not local time. Please rename variable accordingly

Reply:
renamed variable